vm/qemu/qemu.go

```diff
@@ -534,7 +534,26 @@ func splitArgs(str, templateDir string, index int) (args []string) {
 		if arg == "" {
 			continue
 		}
-		arg = strings.ReplaceAll(arg, "{{INDEX}}", fmt.Sprint(index))
+		// 0-7 VFs/Device, if index >= 8, Device# += (index / 8)
+		if strings.Contains(arg, "{{INDEX}}") {
+			arg = strings.ReplaceAll(arg, "{{INDEX}}", fmt.Sprint(index%8))
+			if index > 7 && strings.Contains(arg, "vfio-pci,host=") {
+				// VFs are 0:7. Need to increment the Device if VF >= 8
+				args1 := strings.Split(arg, "vfio-pci,host=")
+				args2 := strings.Split(args1[1], ":")
+				args3 := strings.Split(args2[1], ".")
+				n, err := strconv.ParseInt(args3[0], 16, 32)
+				if err != nil {
+					fmt.Println(err)
```

Comment:
This is not a good way to handle errors. This code shouldn't print and we shouldn't silently ignore the error. We need to return the error to the caller.